src/acom_music_box/music_box.py

```diff
@@ -195,23 +194,7 @@ def solve(self, output_path=None, callback=None):
                 curr_time += time_step
                 pbar.update(time_step)
         df = pd.DataFrame(output_array[1:], columns=output_array[0])
```

Comment:
```suggestion
        return pd.DataFrame(output_array[1:], columns=output_array[0])
```

Let's just return the results directly now